waiter/src/waiter/handler.clj

```diff
@@ -1174,14 +1174,52 @@
   [request]
   (utils/exception->response (ex-info (utils/message :not-found) {:log-level :info :status http-404-not-found}) request))
 
+(defn drain-handler
+  "Handles Get /drain requests by providing the current drain state.
+  Handles Post /drain requests and sets the drain-until for the router based on the provided 'drain-timeout-ms' query
```

Comment:
I feel like this timeout should be in seconds rather than milliseconds.

I can't think of a situation where we'd want milliseconds here, and it seems easy to accidentally set this thinking it's seconds instead, and interrupt all of the active connections.

Reply:
We should reflect the unit in the query parameter name.